pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1642,17 +1642,14 @@
         ::
         =*  have=mark  p.cage.sign
         =*  desc=tape  "from {(trip have)} to json"
-        =/  convert=(unit $-(* json))
+        =/  convert=(unit vase)
           =/  cag=(unit (unit cage))
             (rof ~ %cf [our %home da+now] /[have]/json)
           ?.  ?=([~ ~ *] cag)  ~
-          `!<($-(* json) q.u.u.cag)
+          `q.u.u.cag
         ?~  convert
           ((slog leaf+"eyre: no convert {desc}" ~) [~ ~])
-        =/  res  (mule |.((u.convert q.cage.sign)))
-        ?:  ?=(%& -.res)
-          [`have `[%fact %json !>(p.res)]]
-        ((slog leaf+"eyre: failed tube {desc}" ~) [~ ~])
+        [`have `[%fact %json (slam u.convert q.cage.sign)]]
```

Comment:
Hmm I thought this (the `mule`) was protecting against conversion failing due to `!!` in the implementation, not necessarily clay giving us bogus or w/e. Not sure we're fine with crashing in those cases.

Reply:
@joemfb @belisarius222 Can I have some input here? From profiling with mule vs without, using it with mule wipes out the majority of the performance gain we received from switching to `+slam`.

Reply:
We should think about whether virtualization is really necessary here. Is it better for %eyre to drop facts, or crash the event?

Regardless of the +mule, you can use `+slym` instead of `+slam` to skip nest checks (since we trust %clay).

Reply:
It looks like the current "protection" is just a no-op, which seems like a footgun to me.  If we're going to catch errors, I'd be inclined to let the app know that its attempt to deliver an effect to the browser failed, rather than pretending everything is ok.

Also, try +slym instead of +slam, as @joemfb suggested.

Reply:
@joemfb beat me to it :)

Reply:
For even more juice, you could make this codepath stateful and use `+slym:wa`.

Reply:
I believe crashing here will "just" result in the subscription getting killed, eyre getting the `%kick`, and proceeding from there as normal. I vaguely remember some gotcha in this area, but might have just been aggressive resubscribe behavior on the client side.  
@tacryt-socryp this is probably fine (and better than swallowing conversion errors), but worth testing the "conversion logic crashes" case to make sure nothing borks.